Second commit

wajdm opened the pull request:
I've updated my main page, made new pages (Car, motorcycle, help), with their styles.css. I've also added a scripts.js file for javascript.

wajdm committed:
Delete styles.css

wajdm committed:
Delete index.html

wajdm committed:
Add files via upload

Second commit.

wajdm committed:
Add files via upload

Mr-Coxall commented:
From cars.html I just removed: <script defer src="Scripts/script.js"></
script>
From what I can see, you are actually not using any of the script code, so
no need for the reference.

Also, for style can you rename Scripts directory to scripts, or even better
"js".

Mr. Coxall

On Mon, Jan 25, 2021 at 12:02 PM wajd-mariam <notifications@github.com>
wrote:

> I've updated my main page, made new pages (Car, motorcycle, help), with
> their styles.css. I've also added a scripts.js file for javascript.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2
> Commit Summary
>
>    - Delete styles.css
>    - Delete index.html
>    - Add files via upload
>    - Add files via upload
>
> File Changes
>
>    - *A* Scripts/questions.json
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-7ae17133088e6aab1d59f34765af634a0c190deca0067bb0c9f11f2b66d4a256>
>    (4)
>    - *A* Scripts/script.js
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-cb93a3310e469034ce26d6e9d5a7ffaa365b429920460cac6d32c8731c5b2372>
>    (44)
>    - *A* Styles/main_styles.css
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-41b04548e597239a7e1cecce87180f4d2cbb564ab42f5670329ffc48c7a31f26>
>    (96)
>    - *A* Styles/options_styles.css
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-67195a7ef52af84ce5617e9a6f1cc78bbd8867db0ce04f788a628b40ed770ac5>
>    (40)
>    - *A* Styles/vehicle_styles.css
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-15e3cb2fe10d335503e463401a3e0fcfb7e42fccccc20365e937a2006ffc2976>
>    (105)
>    - *A* car.html
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-a786c9a0f4acb32675023a7bcc4a8626a828f98b8b739d2e34427212a8249787>
>    (36)
>    - *A* help.html
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-8954bd8b685d4d0f3d5c30a93054f9e0c323f916e761dedb785c098532171d8e>
>    (32)
>    - *M* index.html
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051>
>    (71)
>    - *A* motorcycle.html
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-9d27c142e969ecc758147ceef5695130d9ce1eb8486f80931c3ad8255a03c255>
>    (35)
>    - *D* styles.css
>    <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1>
>    (165)
>
> Patch Links:
>
>    - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.patch
>    - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4SJSLCGFH5X5XWU4YWY3TS3WP2ZANCNFSM4WSBY7PQ>
> .
>


-- 
CONFIDENTIALITY NOTICE: This email, including any attachments, is for the
sole use of the intended recipient(s) and may contain confidential and
privileged information. Any unauthorized review, use, disclosure, or
distribution is prohibited. If you received this email and are not the
intended recipient, please inform the sender by email reply and destroy all
copies of the original message.

wajdm commented:
Ok will do. But the issue was with the main page, specifically with the two
buttons there, after "Select your vehicle", there is a "Car" button and a
"Motorcycle" button, when I press any of them I get this error message
"script.js:17 Uncaught TypeError: Cannot read property 'addEventListener'
of null". As for the car.html, I haven't gotten to that point  of editing
yet. Thanks

On Mon., Jan. 25, 2021, 12:31 p.m. Patrick Coxall, <notifications@github.com>
wrote:

> From cars.html I just removed: <script defer src="Scripts/script.js"></
> script>
> From what I can see, you are actually not using any of the script code, so
> no need for the reference.
>
> Also, for style can you rename Scripts directory to scripts, or even better
> "js".
>
> Mr. Coxall
>
> On Mon, Jan 25, 2021 at 12:02 PM wajd-mariam <notifications@github.com>
> wrote:
>
> > I've updated my main page, made new pages (Car, motorcycle, help), with
> > their styles.css. I've also added a scripts.js file for javascript.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2
> > Commit Summary
> >
> > - Delete styles.css
> > - Delete index.html
> > - Add files via upload
> > - Add files via upload
> >
> > File Changes
> >
> > - *A* Scripts/questions.json
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-7ae17133088e6aab1d59f34765af634a0c190deca0067bb0c9f11f2b66d4a256
> >
> > (4)
> > - *A* Scripts/script.js
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-cb93a3310e469034ce26d6e9d5a7ffaa365b429920460cac6d32c8731c5b2372
> >
> > (44)
> > - *A* Styles/main_styles.css
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-41b04548e597239a7e1cecce87180f4d2cbb564ab42f5670329ffc48c7a31f26
> >
> > (96)
> > - *A* Styles/options_styles.css
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-67195a7ef52af84ce5617e9a6f1cc78bbd8867db0ce04f788a628b40ed770ac5
> >
> > (40)
> > - *A* Styles/vehicle_styles.css
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-15e3cb2fe10d335503e463401a3e0fcfb7e42fccccc20365e937a2006ffc2976
> >
> > (105)
> > - *A* car.html
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-a786c9a0f4acb32675023a7bcc4a8626a828f98b8b739d2e34427212a8249787
> >
> > (36)
> > - *A* help.html
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-8954bd8b685d4d0f3d5c30a93054f9e0c323f916e761dedb785c098532171d8e
> >
> > (32)
> > - *M* index.html
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051
> >
> > (71)
> > - *A* motorcycle.html
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-9d27c142e969ecc758147ceef5695130d9ce1eb8486f80931c3ad8255a03c255
> >
> > (35)
> > - *D* styles.css
> > <
> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1
> >
> > (165)
> >
> > Patch Links:
> >
> > - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.patch
> > - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AA4SJSLCGFH5X5XWU4YWY3TS3WP2ZANCNFSM4WSBY7PQ
> >
> > .
> >
>
>
> --
> CONFIDENTIALITY NOTICE: This email, including any attachments, is for the
> sole use of the intended recipient(s) and may contain confidential and
> privileged information. Any unauthorized review, use, disclosure, or
> distribution is prohibited. If you received this email and are not the
> intended recipient, please inform the sender by email reply and destroy all
> copies of the original message.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2#issuecomment-766981729>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDJIIVT66VJFC4YYXIO3LTS3WTFJANCNFSM4WSBY7PQ>
> .
>

wajdm commented:
Never mind sir, I got it all working now, sorry for emailing you again
about the issue, I thought there was a misunderstanding.

Thanks a lot!

On Mon, Jan 25, 2021 at 12:46 PM Wajd Mariam <wajd.mariam@mths.ca> wrote:

> Ok will do. But the issue was with the main page, specifically with the
> two buttons there, after "Select your vehicle", there is a "Car" button and
> a "Motorcycle" button, when I press any of them I get this error message
> "script.js:17 Uncaught TypeError: Cannot read property 'addEventListener'
> of null". As for the car.html, I haven't gotten to that point  of editing
> yet. Thanks
>
> On Mon., Jan. 25, 2021, 12:31 p.m. Patrick Coxall, <
> notifications@github.com> wrote:
>
>> From cars.html I just removed: <script defer src="Scripts/script.js"></
>> script>
>> From what I can see, you are actually not using any of the script code, so
>> no need for the reference.
>>
>> Also, for style can you rename Scripts directory to scripts, or even
>> better
>> "js".
>>
>> Mr. Coxall
>>
>> On Mon, Jan 25, 2021 at 12:02 PM wajd-mariam <notifications@github.com>
>> wrote:
>>
>> > I've updated my main page, made new pages (Car, motorcycle, help), with
>> > their styles.css. I've also added a scripts.js file for javascript.
>> > ------------------------------
>> > You can view, comment on, or merge this pull request online at:
>> >
>> > https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2
>> > Commit Summary
>> >
>> > - Delete styles.css
>> > - Delete index.html
>> > - Add files via upload
>> > - Add files via upload
>> >
>> > File Changes
>> >
>> > - *A* Scripts/questions.json
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-7ae17133088e6aab1d59f34765af634a0c190deca0067bb0c9f11f2b66d4a256
>> >
>> > (4)
>> > - *A* Scripts/script.js
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-cb93a3310e469034ce26d6e9d5a7ffaa365b429920460cac6d32c8731c5b2372
>> >
>> > (44)
>> > - *A* Styles/main_styles.css
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-41b04548e597239a7e1cecce87180f4d2cbb564ab42f5670329ffc48c7a31f26
>> >
>> > (96)
>> > - *A* Styles/options_styles.css
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-67195a7ef52af84ce5617e9a6f1cc78bbd8867db0ce04f788a628b40ed770ac5
>> >
>> > (40)
>> > - *A* Styles/vehicle_styles.css
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-15e3cb2fe10d335503e463401a3e0fcfb7e42fccccc20365e937a2006ffc2976
>> >
>> > (105)
>> > - *A* car.html
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-a786c9a0f4acb32675023a7bcc4a8626a828f98b8b739d2e34427212a8249787
>> >
>> > (36)
>> > - *A* help.html
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-8954bd8b685d4d0f3d5c30a93054f9e0c323f916e761dedb785c098532171d8e
>> >
>> > (32)
>> > - *M* index.html
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051
>> >
>> > (71)
>> > - *A* motorcycle.html
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-9d27c142e969ecc758147ceef5695130d9ce1eb8486f80931c3ad8255a03c255
>> >
>> > (35)
>> > - *D* styles.css
>> > <
>> https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1
>> >
>> > (165)
>> >
>> > Patch Links:
>> >
>> > - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.patch
>> > - https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2.diff
>> >
>> > —
>> > You are receiving this because you are subscribed to this thread.
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2>, or
>> > unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/AA4SJSLCGFH5X5XWU4YWY3TS3WP2ZANCNFSM4WSBY7PQ
>> >
>> > .
>> >
>>
>>
>> --
>> CONFIDENTIALITY NOTICE: This email, including any attachments, is for the
>> sole use of the intended recipient(s) and may contain confidential and
>> privileged information. Any unauthorized review, use, disclosure, or
>> distribution is prohibited. If you received this email and are not the
>> intended recipient, please inform the sender by email reply and destroy
>> all
>> copies of the original message.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/MotherTeresaHS/ICS4U-2020-Wajd_M/pull/2#issuecomment-766981729>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ANDJIIVT66VJFC4YYXIO3LTS3WTFJANCNFSM4WSBY7PQ>
>> .
>>
>
